docs/rules/jsx-no-useless-fragment.md

```diff
@@ -32,6 +32,10 @@ Examples of **incorrect** code for this rule:
 Examples of **correct** code for this rule:
 
 ```jsx
+{foo}
+
+<Foo />
```

Comment:
i'm confused, why are these needed to make the following fragment useless?

Reply:
ah sorry if this was unclear, they were intended to be the simplifications for these (the first two examples in the incorrect code):

```jsx
<>{foo}</>

<><Foo /></>
```